GitHub_Learning/Feature/Login/Presentation/LoginView.swift

```diff
@@ -0,0 +1,14 @@
+//
+//  LoginView.swift
+//  GitHub_Learning
+//
+//  Created by Amin Ashhab on 16/11/2020.
+//  Copyright © 2020 Amin_Second_Test_Project. All rights reserved.
+//
+
+import Foundation
+import CleanCore
+
+protocol LoginView: View {
+    func isLoginEnabled(_ enabled: Bool)
```

Comment:
Pouzivame spis isEnabled: Bool. Takze:
```suggestion
    func isLoginEnabled(_ isEnabled: Bool)
```